source/plugins/go/src/oms.go

```diff
@@ -502,6 +520,46 @@ func populateExcludedStderrNamespaces() {
 	}
 }
 
+func populateIncludedStdoutSystemResource() {
+	collectStdoutLogs := os.Getenv("AZMON_COLLECT_STDOUT_LOGS")
+	var stdoutIncludedSystemResourceList []string
+	includeList := os.Getenv("AZMON_STDOUT_INCLUDED_SYSTEM_PODS")
+	if (collectStdoutLogs == "true") && (len(includeList) > 0) {
+		stdoutIncludedSystemResourceList = strings.Split(includeList, ",")
+		for _, resource := range stdoutIncludedSystemResourceList {
+			colonLoc := strings.Index(resource, ":")
+			if colonLoc == -1 {
+				Log("Skipping invalid system resource namespace:controller %s for stdout log collection", resource)
+				continue
+			}
+			namespace := resource[:colonLoc]
+			Log("Including system resource namespace:controller %s for stdout log collection", resource)
+			StdoutIncludeSystemResourceSet[strings.TrimSpace(resource)] = true
+			StdoutIncludeSystemNamespaceSet[strings.TrimSpace(namespace)] = true
+		}
+	}
+}
+
+func populateIncludedStderrSystemResource() {
```

Comment:
since the logic common for both stderr and stdout. I think, we should merge both these functions populateIncludedStdoutSystemResource & populateIncludedStderrSystemResource.